src/overlays/actors/ovl_En_Trt/z_en_trt.c

```diff
@@ -1,15 +1,92 @@
+/*
+ * File: z_en_trt.c
+ * Overlay: ovl_En_Trt
+ * Description: Kotake
+ */
+
 #include "z_en_trt.h"
 
 #define FLAGS 0x00000009
 
 #define THIS ((EnTrt*)thisx)
 
+#define ENTRT_FULLY_AWAKE 1
```

Comment:
Why are these defines rather than an emum?

Reply:
They are used as bit flags, which throughout the repo seem to be defines instead of enums.

Reply:
Then perhaps using `(1 << 0)`, `(1 << 1)`, and `(1 << 2)` to emphasize that they are bitflags?